src/item_factory.cpp

```diff
@@ -191,6 +198,12 @@ void Item_factory::finalize_pre( itype &obj )
         }
     }
 
+    // Set max volume for containers to prevent integer overflow
+    if( obj.container && obj.container->contains > units::from_milliliter( 10000000 ) ) {
```

Comment:
I think you can use the user-defined literal for these constants; so this would be `10000000_ml`.

Reply:
Yup, Tried it and it works the same, thanks!